CMakeLists.txt

```diff
@@ -152,7 +153,8 @@ include_directories(
         deps-build/${PLATFORM}/zlib/include
         deps-build/${PLATFORM}/aws/include
         include
-        lib)
+        lib
+        cpp/include)
```

Comment:
Is this an internal header directory?

Reply:
This is a header directory that the user needs to include when using the C++ library.

Reply:
If external headers, please move them to `include`.